src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -210,18 +267,23 @@ const HotspotDetails = () => {
                 })}
               </Text>
             </Box>
-            <TimelinePicker onTimelineChanged={onTimelineChanged} />
+            <TimelinePicker
+              index={timelineIndex}
+              onTimelineChanged={onTimelineChanged}
+            />
             <HotspotDetailChart
```

Comment:
![iPhone 11 2021-01-27 at 8 58 40 AM](https://user-images.githubusercontent.com/1760821/106009478-002c9680-607e-11eb-83ab-5740ced4bb76.jpg)
If I set the picker to "Past 7 Days" my HNT Rewards chart looks like something is not right.

Reply:
Ok I think its better now